packages/l/libredwg/xmake.lua

```diff
@@ -0,0 +1,19 @@
+package("libredwg")
+    set_homepage("https://github.com/LibreDWG/libredwg")
+    set_description("This is a library to allow reading data from a DWG file. That's a very important acquisiton, which may improve a lot the ability of the free software comunity to develop more features in the field of computer technical drawing (CAD). ")
+
+    add_urls("https://github.com/LibreDWG/libredwg.git")
+    add_versions("0.13.3", "97c7225596c17430b82fd0161e7eff6beb5b1034")
```

Comment:
Why not use https://github.com/LibreDWG/libredwg/archive/refs/tags/0.13.3.tar.gz

Reply:
because it use jsmn as submodule, and jsmn is not updated for a long time, I think xmake does not have this package at first🤣

Reply:
fixed

Reply:
https://github.com/xmake-io/xmake-repo/actions/runs/8260691010/job/22596637936?pr=3494
looks like this package does not like external dependency.

Reply:
So I still choose to use the original version.